test_runner/performance/pageserver/pagebench/test_ondemand_download_churn.py

```diff
@@ -0,0 +1,163 @@
+import json
+from pathlib import Path
+from typing import Any, Dict, Tuple
+
+import pytest
+from fixtures.benchmark_fixture import MetricReport, NeonBenchmarker
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv, NeonEnvBuilder, PgBin, wait_for_last_flush_lsn
+from fixtures.utils import get_scale_for_db, humantime_to_ms
+
+from performance.pageserver.util import (
+    setup_pageserver_with_tenants,
+)
+
+
+@pytest.mark.parametrize("duration", [30])
+@pytest.mark.parametrize("pgbench_scale", [get_scale_for_db(200)])
```

Comment:
I have tried just putting 200 without `get_scale_for_db` but those test cannot pass since it takes a lot of time for tenant to becomes active and `ensure_pageserver_ready_for_benchmarking` fails